### DISCUSSION TITLE
how to setup newsletter?

### DISCUSSION THREAD

**Discussion by thinker732** [thelovekesh/ghost-v4-on-heroku#12] [Q&A]
i setup mail and  i test with my mail and it's working 
but newsletter and subscription doesn't work for another mail
how to setup that please?

**Comment by thelovekesh** [comment]
To get started with newsletter and subscriptions you have to configure some SMTP credentials in the Ghost Admin Panel itself. Check for the same in the settings and if it doesn't work.. let me know with some screenshots.

**Reply by thinker732** [reply]
in settings?
i mean for the ghost admin panel 
i setup mailgun and others so it's okay but what do you mean about 
settings?

**Comment by thinker732** [comment]
![github](https://user-images.githubusercontent.com/59264884/126989474-104d0826-858c-42c6-b4a4-8fc09b9d8d1f.png)


this is what i see when i try to newsletter an email that doesn't register in mailmgun 
you can also try to register in the website you'll see it's doesn't work
https://alors-on-dev.herokuapp.com/
